Patches/PlayerControlPatch.cs

```diff
@@ -128,6 +128,7 @@ public static bool Prefix(PlayerControl __instance, [HarmonyArgument(0)] PlayerC
 
         // Is the target in a killable state?
         if (target.Data == null // Check if PlayerData is not null
+            || target.PlayerId == killer.PlayerId // Impossible interaction
             // Check target status
```

Comment:
I think Sniper code says otherwise

![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/8cb095e8-a3f9-40c8-adde-b96ad0c2ba82)

Reply:
Then it should not use check murder